exercises/largest-series-product/largest_series_product.py

```diff
@@ -1,2 +1,2 @@
-def largest_product():
+def largest_product(digits):
```

Comment:
According to test cases https://github.com/exercism/python/blob/master/exercises/largest-series-product/largest_series_product_test.py `largest_product` takes two parameters

Please correct your suggestion

Reply:
Pushed 6714093 with the change. Thanks!